match/lib/match/options.rb

```diff
@@ -223,6 +223,14 @@ def self.available_options
                                      env_name: "MATCH_S3_OBJECT_PREFIX",
                                      description: "Prefix to be used on all objects uploaded to S3",
                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :s3_endpoint,
+                                     env_name: "MATCH_S3_ENDPOINT",
+                                     description: "Custom endpoint to be used for S3",
+                                     optional: true),
+        FastlaneCore::ConfigItem.new(key: :s3_force_path_style,
+                                     env_name: "MATCH_S3_FORCE_PATH_STYLE",
+                                     description: "Force path style for S3",
```

Comment:
Can you be more descriptive here? As someone who is not familiar with this, it's hard to get a grasp of what this does, when to use, etc 🙇